### ISSUE TITLE
Spotify Web Player doesn't load

### ISSUE THREAD

**Issue by anzpanz** [Faerbit/persistentpin-webextension#12]
When starting the browser most of the time Spotify Web Player (https://open.spotify.com/) doesn't load anything and just gives blank page (doesn't even load the favicon).

**Comment by Faerbit** [comment]
I noticed the same thing with my homepage. I can't recall that this happened very recently, so it might have been fixed.
This looks like a firefox bug to me. What version are you using?

**Comment by anzpanz** [comment]
> 
> 
> I noticed the same thing with my homepage. I can't recall that this happened very recently, so it might have been fixed.
> This looks like a firefox bug to me. What version are you using?

Actually I'm not sure if it has happened today at all. The thing is, I moved from Chrome to Firefox yesterday so the extension was a first time install and I also had to log in to Spotify again etc. during all that - maybe the problem was related to that specific environment. And maybe a reboot or cookies or something has fixed it.. But I'll monitor this and see if it still occurs.

I'm using just the official standard release of Firefox, it seems to be version 80.0 64-bit.
